gnome-extra/gnome-commander/gnome-commander-1.14.0.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+GNOME2_LA_PUNT="yes"
+
+inherit gnome2 optfeature
+
+DESCRIPTION="A graphical, full featured, twin-panel file manager"
+HOMEPAGE="https://gcmd.github.io/"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="chm exif gsf pdf samba taglib test +unique"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-text/yelp-tools
+	dev-libs/atk
+	dev-libs/glib:2
+	gnome-base/gconf:2
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/harfbuzz:=
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/pango
+	chm? ( dev-libs/chmlib )
+	exif? ( >=media-gfx/exiv2-0.14 )
+	gsf? ( gnome-extra/libgsf:= )
+	pdf? ( >=app-text/poppler-0.18 )
+	samba? ( gnome-base/gvfs[samba] )
+	taglib? ( >=media-libs/taglib-1.4 )
+	unique? ( >=dev-libs/libunique-0.9.3:1 )
+"
+BDEPEND="
+	${RDEPEND}
```

Comment:
`RDEPEND` make no sense in `BDEPEND`.

Reply:
Hm.... It was like that also in previous versions. I can remove it and check again.

Reply:
I mentioned the problem here: https://github.com/gentoo/gentoo/pull/20619#discussion_r624756558

For whatever reason this slipped through on pushing the PR unfortunately. Most likely the code was changed and the original change request was no longer visible to me as a consequence.

Reply:
Oha. Got it, thanks. Will change accordingly.

Reply:
Updated. Well, I only can admit that it slipped through my fingers too... °_°